lib/signaling/v2/renderhintssignaling.js

```diff
@@ -85,13 +85,13 @@ class RenderHintsSignaling extends MediaSignaling {
   setTrackHint(trackSid, renderHint) {
     // convert hint to msp format
     const mspHint = {
-      'track_sid': trackSid,
+      'track': trackSid,
```

Comment:
Should we just keep this as `track_sid` since it's really a track's sid?

Reply:
it's not really up to us. It was decided collectively to use `track` since thats what we use in rest of the MSP protocols. VMS is doing the same change on server side.